docs/docs/sample-scripts/spo/remove-api-permissions.md

```diff
@@ -0,0 +1,38 @@
+# Remove SharePoint API permissions
```

Comment:
Let's add a reference to the script in mkdocs.yml so that the script will be included in the generated docs.